data_steward/curation_logging/curation_gae_handler.py

```diff
@@ -1,27 +1,32 @@
+"""In GAE py2 request logs are automatically correlated to app logs in StackDriver when using the standard logging
+libraries. This is no longer the case in GAE py3. Although GAE py3 provides the stderr logger by default,
+it prints every single line as a separate long entry, in particular the query logs span multiple log entries in
+StackDriver logger, making it really hard to find anything. The RDR team informed us that we’d need to configure our
```

Comment:
nit: I would drop the historical stuff here and keep it to just statements of fact, i.e. would just drop this sentence "The RDR team informed...", and change the next sentence "Based closely on https://github.com/all-of-us/raw-data-repository/blob/1.60.6/rdr_service/services /gcp_logging.py."

Reply:
Thanks for the suggestion, this is done.